sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -1513,8 +1499,20 @@ SELECT
   map_sum_aggregates[OFFSET(35)].map AS scalar_parent_urlbar_picked_topsite_sum,
   map_sum_aggregates[OFFSET(36)].map AS scalar_parent_urlbar_picked_unknown_sum,
   map_sum_aggregates[OFFSET(37)].map AS scalar_parent_urlbar_picked_visiturl_sum,
-  map_sum_aggregates[OFFSET(38)].map AS search_with_ads,
-  map_sum_aggregates[OFFSET(39)].map AS ad_clicks,
+  CASE
+    WHEN mozfun.norm.truncate_version(app_version, "major") <= 108
+      THEN map_sum_aggregates[OFFSET(38)].map
+    WHEN mozfun.norm.truncate_version(app_version, "major") <= 108
```

Comment:
```suggestion
    WHEN mozfun.norm.truncate_version(app_version, "major") > 108
```
?

Reply:
I would use IF here:
```
IF(
  mozfun.norm.truncate_version(app_version, "major") > 108,
  map_sum_aggregates[OFFSET(113)].map,
  map_sum_aggregates[OFFSET(38)].map
) AS search_with_ads
```

Reply:
On third thought, can we do this in the view? Less likelihood of problems

Reply:
That seems reasonable to me, with 2 caveats:

1. `clients_first_seen` and `clients_last_seen` pull directly from the table here, so they're not going to pick up the new logic for these two fields. I don't have a super great understanding of how `clients_first_seen` and `clients_last_seen` are used, but should we also change the views for those 2 tables?
   
   For that matter, is there a reason that those tables query this table directly and not the view?

2. Is there a more descriptive name we can use to refer to these new fields and make it clear that they are the versions coming from the new probe? @mmccorks any thoughts?

Reply:
Oh and thanks for catching that!

Reply:
Thanks Frank. 
Curtis - I agree that we want `clients_first_seen` and `clients_last_seen` to also have the data from the new probes. If that means updating those views as well, that works. I'm fine with doing this in the view if that makes the most sense.  
For the filed names - I don't know that we want to change the name in the risk of breaking existing queries against this table, but we could add a note in the analysis gotchas and update the documentation for this table on DTMO to make sure that this change is documented properly.